docs/reference/commandline/start.md

```diff
@@ -22,6 +22,85 @@ Start one or more stopped containers
 
 ## Examples
 
+### Start a container
+
+To start a container, specify the container ID or image name. For example:
+
 ```console
 $ docker start my_container
 ```
+
+### <a name="attach"></a>  Start a container and attach to its STDOUT/STDERR (--attach, -a)
+
+To start a container and attach to its STDOUT/STDERR and forward signals, use the `--attach` or `-a` option. For example, if you create an nginx container named my_nginx_container, you can start it and monitor its logs using the following:
+
+```console
+$ docker create --name my_nginx_container -p 80:80 nginx
+ca5a4351c9c4b43c0b0c69d4d925aa1d8a53a3b33742250ad227f22096accab6
+
+$ docker start -a my_nginx_container
+/docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
+/docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
+/docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
+...
+```
+
+### <a name="checkpoint"></a> Start a container and restore from a checkpoint (--checkpoint) (experimental)
+
+The `--checkpoint` option is an experimental feature, and should not be
+considered stable. To read about experimental daemon options and how to enable
+them, see
+[Daemon configuration file](https://docs.docker.com/engine/reference/commandline/dockerd/#daemon-configuration-file).
+
+To start a container and restore it from a checkpoint, use the `--checkpoint` option. For example:
+
+```console
+$ docker run --security-opt=seccomp:unconfined --name cr -d busybox /bin/sh -c 'i=0; while true; do echo $i; i=$(expr $i + 1); sleep 1; done'
+db316bda7d64b4154d207a3d659c90c982d0b35a3e177fc396e809a6a93a147f
+
+$ docker checkpoint create cr checkpoint1
+checkpoint1
+
+# <later>
+$ docker start --checkpoint checkpoint1 cr
```

Comment:
I don't think we need the `--security-opt` for this example. Perhaps we should give the container a more descriptive name (`my_container ` e.g.);

```suggestion
$ docker run --name my_container -d busybox /bin/sh -c 'i=0; while true; do echo $i; i=$(expr $i + 1); sleep 1; done'
db316bda7d64b4154d207a3d659c90c982d0b35a3e177fc396e809a6a93a147f

$ docker checkpoint create my_container checkpoint1
checkpoint1

# <later>
$ docker start --checkpoint checkpoint1 my_container
```